src/arm.rs

```diff
@@ -32,63 +32,57 @@ pub unsafe extern "aapcs" fn __aeabi_uldivmod() {
     intrinsics::unreachable();
 }
 
-extern "C" {
-    fn memcpy(dest: *mut u8, src: *const u8, n: usize) -> *mut u8;
-    fn memmove(dest: *mut u8, src: *const u8, n: usize) -> *mut u8;
-    fn memset(dest: *mut u8, c: i32, n: usize) -> *mut u8;
-}
-
```

Comment:
I think it is better to leave this as it was since the `memcpy` symbol might be overridden.